### PR TITLE
fix: update platform to 9.0 in podspec.

### PR DESCRIPTION
**Pull request by wanhmr**
A compile error  occurs when using .interruptSpokenAudioAndMixWithOthers.

### CODE REVIEW

File: packages/audioplayers_darwin/ios/audioplayers_darwin.podspec [hunk 1]
```diff
@@ -13,7 +13,7 @@ Pod::Spec.new do |s|
   s.source           = { :path => '.' }
   s.source_files = 'Classes/**/*'
   s.dependency 'Flutter'
-  s.platform = :ios, '8.0'
+  s.platform = :ios, '9.0'
```
Comment (luanpotter) [inline packages/audioplayers_darwin/ios/audioplayers_darwin.podspec:16]: will that prevent this from working on older devices? I recall we had issues before when dropping support

**Comment by spydon** [comment]
@wanhmr just repeating luans question to see whether we can merge this or not; will this drop support for older devices?

**Comment by wanhmr** [comment]
We can only use .interruptSpokenAudioAndMixWithOthers in iOS9.

**Comment by Gustl22** [comment]
Given that ios 14 only has 2% share on ios devices.
I think we can easily bump to ios 9 with no consequences (most likely below 1%).

https://gs.statcounter.com/ios-version-market-share/mobile-tablet/worldwide/#monthly-202108-202208-bar

**Review by spydon — APPROVED** [review]
I spoke to Luan and he approves too.
Let's just hope it doesn't create a bigger backlash than it's worth.

**Pull request merged**